prod/westeurope/common/subnet_private_endpoints/terragrunt.hcl

```diff
@@ -0,0 +1,40 @@
+#  Subnet to place all private endpoints
```

Comment:
The name of the directory must be subnet_pendpoints instead of subnet_private_endpoints.